caddytest/integration/caddyfile_adapt/templates.txt

```diff
@@ -0,0 +1,44 @@
+
+----------
+{
+  "admin": {
+    "listen": "127.0.0.1:8081"
+  },
```

Comment:
Remove this file; the test isn't correct, look at other adapt tests. The format is a Caddyfile at the start, then a `----------` 10 dash separator, and then the JSON adapt output.

In this case, you're just adding a new function to the templates, which isn't related to the Caddyfile